### ISSUE TITLE
Implement a libslic3r-based slicing

### ISSUE THREAD

**Issue by LubosD** [dashprint/dashprint#4]
Fully running in the browser via webassembly.

**Comment by DeepBhat** [comment]
Hey LubosD, I was curious about this project. Did you manage to compile libslic3r to webassembly?

**Comment by LubosD** [comment]
No, I gave up on it. It didn't seem to be that difficult, although the code was written with no regard to systems without threading (it's still better to avoid them in wasm).

My original plan was to eventually create a cloud management platform for printers with this and OpenSCAD functionality included, but I don't feel like competing with the likes of PrusaConnect.
